### ISSUE TITLE
Please add detailed description to Readme

### ISSUE THREAD

**Issue by sleepycat** [keokilee/react-electron-webview#5]
This project would really benefit from some details in the readme explaining what problem it solves and how it differs from other projects like `react-electron-web-view`.

The standard Installation/Usage instructions would be great as well.

**Comment by keokilee** [comment]
Apologies for the lack of README. I originally open-sourced this while working on a Electron project (which I haven't touched in a long time), so I didn't really put much thought into a README. I see that project that you referenced is much newer than this one, so really the main difference is that this one is older 😅 . In any case, it's cool that people are checking the project out, and it definitely needs a README.

**Comment by archae0pteryx** [comment]
I totally agree. You've got all the props. As far as i have read this project is a bit more 'full-featured' than ```react-electron-web-view``` You should create a useful readme. Just an opinion from a fan. =) cheers.
